e2e-tests/tests/features/dbengine_upgrade/31-assert.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
+timeout: 300
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseEngine
+metadata:
+  name: percona-xtradb-cluster-operator
+spec:
+  type: pxc
+status:
+  status: installed
+  operatorVersion: 1.13.0
+
```

Comment:
Should we also assert that the annotation has been removed?

Reply:
Sure, updated :)